src/Screens/Onboarding/SignUp.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import RN from 'react-native';
+import { SafeAreaView } from 'react-navigation';
+
+import * as navigationProps from '../../lib/navigation-props';
+
+import Background from '../components/Background';
+import Card from '../components/Card';
+import fonts from '../components/fonts';
+import Message from '../components/Message';
+import colors, { gradients } from '../components/colors';
+import InputField from '../components/InputField';
+import Button from '../components/Button';
+
+export type SignUpRoute = {
+  'Onboarding/SignUp': {};
+};
+
+type OwnProps = navigationProps.NavigationProps<SignUpRoute, SignUpRoute>;
+
+const SignUp = (props: OwnProps) => {
+  const goBack = () => {
+    props.navigation.goBack();
+  };
```

Comment:
more tacit style does not work with react navigation. at least it did not work some months ago : (